tests/async/tfuturevar.nim

```diff
@@ -1,7 +1,6 @@
 discard """
```

Comment:
if action is `run`, it's implied, so the whole discard section can go away (unless you need other testament spec fields)